api/src/test/java/org/cardanofoundation/rosetta/api/construction/service/PayloadsApiTest.java

```diff
@@ -120,4 +137,17 @@ private void assertConstructionPayloads(String requestFile, String expectedUnsig
     assertEquals(expectedUnsignedTransaction, body.getUnsignedTransaction());
   }
 
+  private void assertConstructionPayloadsFail(String requestFile, String expectedErrorMessage) throws Exception {
+    ConstructionPayloadsRequest request = getPayloadRequest(requestFile);
+
+    ApiException exception = Assertions.assertThrows(
+            ApiException.class,
+            () -> constructionApiService.constructionPayloadsService(request)
+    );
+
+    System.out.println(exception.getMessage());
+
+    assertEquals(expectedErrorMessage, exception.getMessage());
```

Comment:
I would rather verify error code because verifiy message error is prone to translation errors (I had this on one of my machines) for different locale.